core/server/data/importer/importers/data/index.js

```diff
@@ -30,24 +30,28 @@ DataImporter = {
         return importData;
     },
 
-    doImport: function doImport(importData) {
-        var ops = [], errors = [], results = [], options = {
+    doImport: function doImport(importData, options) {
```

Comment:
It seems to me that these changes are not related to the new backup endpoint? I'm not sure what they do or how to test them.